test-setup.js

```diff
@@ -1,4 +1,5 @@
 require('raf/polyfill');
+require('mutationobserver-shim');
```

Comment:
MutationObserver is used nowhere in the tests, why did you add this?

Reply:
mutationobserver is used in quilljs

Reply:
i will put in some comments on the ones that require due to quilljs

Reply:
Then the polyfill should be added by quilljs themselves, or am I missing something?

Reply:
i think in default jest uses jsdom and jsdom do not have support for mutationobserver.

https://github.com/jsdom/jsdom/issues/639

Reply:
Right, so you should add that comment to the code so that other developers know why it's included

Reply:
done.